tests/RestApiSuite/TestGetAttributeFromLastReponse/content.txt

```diff
@@ -0,0 +1,7 @@
+!|post    |${URL}/users        |
+|FirstName|LastName|IsMafia|Age|
+|Tony     |Soprano |True   |22 |
+
+-!|script|RestTools|
+|check|get attribute|LastName|from last response|Soprano|
```

Comment:
This technique allows to store created object id in variable
`|$objectId=|get attribute|objectId|from last response|`